Jenkinsfile

```diff
@@ -63,91 +69,146 @@ pipeline {
           // This build is a little unusual; it actually has _two_ artifacts, this one
           // and one for store-pg-assembly.  sbtbuild isn't really set up to override names
           // for two separate artifacts, so here we override the primary, and we'll override
-          // the secondary down below when it's passed to dockerize.
+          // the secondary down below when it's passed to docker.
           sbtbuild.setSrcJar("soql-server-pg/target/soql-server-pg-assembly.jar")
           env.STORE_PG_ARTIFACT = "store-pg/target/store-pg-assembly.jar"
           sbtbuild.build()
         }
       }
     }
-    stage('Dockerize SoQL Server PG') {
+    stage('Docker Build') {
```

Comment:
I've added a top level stage with two nested stages so they can share the `when` logic and the `post` logic.